app/flask_config.py

```
@@ -40,7 +40,7 @@ class BaseConfig:
     MAIL_PASSWORD = Config.MAIL_PASSWORD
 
     # CORS
-    CORS_ORIGINS = [Config.MAIL_HOST]
+    CORS_ORIGINS =  [Config.MAIL_HOST, "http://localhost:3000"]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)